poi-ooxml/src/main/java/org/apache/poi/xwpf/usermodel/XWPFSDTCell.java

```diff
@@ -40,4 +42,7 @@ public ISDTContent getContent() {
         return cellContent;
     }
 
+    public CTSdtCell getSdtCell() {
```

Comment:
Javadoc and `@since` - Include the CT in the method name to highlight that users are heading into generated XMLBean class territory.